.travis.yml

```diff
@@ -1,25 +1,30 @@
 language: php
 
-matrix:
-    include:
-        - php: hhvm
-        - php: 5.3
-        - php: 5.4
-        - php: 5.5
-        - php: 5.6
-        - php: nightly
-    allow_failures:
-        - php: hhvm
-        - php: nightly
-    fast_finish: true
-
 sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
 
-install: composer install -n
+matrix:
```

Comment:
To follow the same order as symfony/symfony did (I took inspiration from their travis.yml), but it's not important. I can rechange it, but I changed the indentation that wasn't the same everywhere already so the diff will be messed up anyway.